src/main/java/com/targetindia/DigitizeTimeTable/controller/AdminController.java

```diff
@@ -0,0 +1,132 @@
+
+package com.targetindia.DigitizeTimeTable.controller;
+
+import com.targetindia.DigitizeTimeTable.model.ClassInfo;
+import com.targetindia.DigitizeTimeTable.model.Instructor;
+import com.targetindia.DigitizeTimeTable.model.Course;
+import com.targetindia.DigitizeTimeTable.service.AdminService;
+//import org.mockito.internal.matchers.Null;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("admins")
+public class AdminController {
+    @Autowired
+    AdminService service;
+
+    @PostMapping("/instructors")
+    public ResponseEntity<String> addInstructor(@RequestBody Instructor instructor)
+    {
+        service.addInstructor(instructor);
+        return ResponseEntity.status(HttpStatusCode.valueOf(201)).body("Instructor added successfully");
+    }
+    @PutMapping("/instructors/{id}/{name}/{contact}")
+    public ResponseEntity<String> updateInstructor(@PathVariable int id, @PathVariable String name, @PathVariable long contact) {
+        String result = service.updateInstructor(id, name, contact);
+        if (result == null) {
+            return ResponseEntity.status(HttpStatusCode.valueOf(404)).body("No instructor found with ID = "+ id + " to update.");
```

Comment:
HttpStatusCode.valueOf(404) should be replaced with HttpStatus.NOT_FOUND for readability. same for 201 with HttpStatus.CREATED